src/display/api.js

```diff
@@ -299,6 +302,9 @@ PDFJS.getDocument = function getDocument(src,
     source = src;
   }
 
+  // Default range chunk size to 2^16 = 65536
+  source.rangeChunkSize = source.rangeChunkSize || 65536;
```

Comment:
Make 65536 a constant (at least justbefore this statement).

Can you use `params` object (instead of `source`) -- we don't want to mutate the original parameter?